web/modules/custom/bnf/src/Plugin/FieldTypeTraits/DateTimeTrait.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\bnf\Plugin\FieldTypeTraits;
```

Comment:
We need to move these, there's no `FiledTypeTraits` plugin in Drupal.

I'd suggest a simple `Drupal\bnf\Plugin\bnf_mapper\Traits` to keep the traits close to the mappers.